### ISSUE TITLE
Recaptcha Component Validation

### ISSUE THREAD

**Issue by javikalsan** [cake17/cakephp-recaptcha#1]
Hi,
i tested Recaptcha Component under cakephp3 enviroment and the Validation not works if Recaptcha is not checked, acording to the actual code on RecaptchaComponent.php : 

if (isset($controller->request->data["g-recaptcha-response"]) && !empty($controller->request->data["g-recaptcha-response"])) { 
 ... Validation

if $controller->request->data["g-recaptcha-response"] is empty, for example, if Recaptcha is not checked the validation does not occur and the POST data is sending to the controller normally.

Adding this lines at top of the function is working for me:

if( empty($controller->request->data["g-recaptcha-response"]))
          {
            $controller->Flash->error( __d('recaptcha', 'Please check Recaptcha Box.'), ['key' => 'alert/error']);
            return $controller->redirect($controller->referer());
          }

Basically we have to control this casuistry also.
Cheers

**Comment by cake17** [comment]
Hi,
Thanks for your message, you're right, this needs a correction.
Would it be possible to you to send a pull request with the changes?

**Comment by cake17** [comment]
close this as it's corrected in #2
